js/common/bridge.go

```diff
@@ -134,6 +134,9 @@ func BindToGlobal(rt *goja.Runtime, data map[string]interface{}) func() {
 }
 
 // Bind the provided value v to the provided runtime
+//
+// Deprecated: JS modules can implement the modules.VU interface for getting the context,
+// goja runtime and the VU State, so the goja.Runtime.Set method can be used for data binding.
 func Bind(rt *goja.Runtime, v interface{}, ctxPtr *context.Context) map[string]interface{} {
```

Comment:
Should we log from `common.Bind` considering that we still use it (init context)?

Reply:
We can even emit a nice warning here, since we have the module name, something like `Module 'k6/x/whaever' is using deprecated APIs that will be removed in k6 v0.38.0, see https://k6.io/docs/extensions/new-api-info for more details on how to update it.` :man_shrugging: